store/sqlstore/channel_store.go

```diff
@@ -298,7 +298,8 @@ func (s SqlChannelStore) CreateIndexesIfNotExists() {
 	s.CreateIndexIfNotExists("idx_channelmembers_channel_id", "ChannelMembers", "ChannelId")
 	s.CreateIndexIfNotExists("idx_channelmembers_user_id", "ChannelMembers", "UserId")
 
-	s.CreateFullTextIndexIfNotExists("idx_channels_txt", "Channels", "Name, DisplayName")
+	s.RemoveIndexIfExists("idx_channels_txt", "Channels")
```

Comment:
Can removing the old index be done as part of the upgrade process in `store/sqlstore/upgrade.go` to keep it with the other changes to existing database objects?